### DISCUSSION TITLE
Web-Design

### DISCUSSION THREAD

**Discussion by Ethan-Bit-256** [mapeditor/tiled#3069] [Q&A]
Yo, Is it possible if I can contribute to the development of the website? I'm looking to extend my skills!

**Comment by bjorn** [comment]
Right, so I only recently rewrote the website and I'm quite happy with how it turned out, but I'll gladly take any suggestions for improvement! 

The website lives in this repository on the `gh-pages` branch and is based on Jekyll. Normally it's quite easy to get started by installing the required components by running `bundle` and you can start a local server using `./_source/server.sh` (which uses Bundler).

**Reply by Ethan-Bit-256** [reply]
Cool!
